tests/notificationprofile/V1/test_views.py

```diff
@@ -53,17 +57,18 @@ def teardown(self):
 
     def test_incidents_filtered_by_notification_profile_view(self):
         response = self.user1_rest_client.get(
-            reverse("v1:notification-profile:notificationprofile-incidents", args=[self.notification_profile1.pk])
+            f"/api/v1/notificationprofiles/{self.notification_profile1.pk}/incidents/"
         )
```

Comment:
This is actually filtered by the time slot pk, which for now is the same as the notification profile pk, but since that is planned to change, this test needs to be changed accordingly.

```suggestion
    def test_get_incidents_filtered_by_timeslot_returns_correct_incidents(self):
        response = self.user1_rest_client.get(
            f"/api/v1/notificationprofiles/{self.notification_profile1.timeslot.pk}/incidents/"
        )
```